src/core/fonts.js

```diff
@@ -5326,15 +5326,21 @@ var Type1Parser = (function Type1ParserClosure() {
   }
 
   function decrypt(data, key, discardNumber) {
-    var r = key | 0, c1 = 52845, c2 = 22719;
-    var count = data.length;
+    if (discardNumber > data.length) {
+      return new Uint8Array(0);
+    }
+    var r = key | 0, c1 = 52845, c2 = 22719, i = 0, j = 0;
```

Comment:
You already initialize `i` and `j` in the loops, so we can remove the initialization here.